framework/src/modules/chain/blocks/block.js

```diff
@@ -120,6 +120,81 @@ const getBytes = block => {
 	return byteBuffer.toBuffer();
 };
 
+const getBytesV1 = block => getBytesV0(block);
+
+/**
+ * Description of the function.
+ *
+ * @param {block} block
+ * @throws {Error}
+ * @returns {!Array} Contents as an ArrayBuffer
+ * @todo Add description for the function and the params
+ */
+const getBytesV2 = block => {
+	const capacity =
+		4 + // version (int)
+		4 + // timestamp (int)
+		4 + // height (int)
+		4 + // maxHeightPreviouslyForged (int)
+		4 + // prevotedConfirmedUptoHeight (int)
```

Comment:
The LIP suggest that these new properties to be added after `previousBlock`, not before it.

https://github.com/LiskHQ/lips/blob/master/proposals/lip-0014.md#additional-block-header-properties

> The properties `height`, `heightPrevious`, `heightPrevoted` should be included in the byte array in this order directly after the `previousBlock` property.